### PR TITLE
fix window undefined check

### LINKED ISSUE #123: window is undefined on server side
https://github.com/JedWatson/react-input-autosize/blob/18842d386f5fbad23c888bf45cb2b9e1f29e0010/src/AutosizeInput.js#L39

Unfortunately it leads to error on server-side..

### PR DESCRIPTION
**Pull request by rkkautsar**
Fixes #123 
If window is undefined, window.document will throw an error

### CODE REVIEW

**Comment by rkkautsar** [comment]
introduced in #121, need to add tests?

**Comment by jurvi** [comment]
Please, approve this ASAP.

**Comment by emolchanov** [comment]
@CallumJFraser @JedWatson pls, merge this in master and release version 2.2.1 on package ASAP

**Comment by CallumJFraser** [comment]
I don't have the ability to merge.

File: src/AutosizeInput.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const copyStyles = (styles, node) => {
 	node.style.textTransform = styles.textTransform;
 };
 
-const isIE = (typeof window === 'undefined' && window.document) ? false : /MSIE |Trident\/|Edge\//.test(window.navigator.userAgent);
+const isIE = (typeof window === 'undefined' || typeof window.document === 'undefined') ? false : /MSIE |Trident\/|Edge\//.test(window.navigator.userAgent);
```
Comment (fsarron-nexway) [inline src/AutosizeInput.js]: maybe typeof window === 'undefined' || typeof window.navigator === 'undefined' ?

**Comment by cscleison** [comment]
for those having CI tests failing, a workaround:

package.json
```
"jest": {
    "globals": {
      "window": {
        "navigator": {}
      }
    }
}
```

**Comment by JedWatson** [comment]
Thanks for the quick fix, I'll publish this asap

**Comment by JedWatson** [comment]
By the way, I would be very happy to add tests for SSR if someone wants to contribute them

**Comment by emolchanov** [comment]
@JedWatson 👍 thanks

**Pull request merged**